openstack_controller/tests/conftest.py

```diff
@@ -0,0 +1,32 @@
+# (C) Datadog, Inc. 2019
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+import os
+
+import pytest
+
+from datadog_checks.dev.ssh_tunnel import socks_proxy
+from datadog_checks.dev.terraform import terraform_run
+from datadog_checks.dev.utils import get_here
+
+
+@pytest.fixture(scope='session')
+def dd_environment():
+    with terraform_run(os.path.join(get_here(), 'terraform')) as outputs:
+        if not outputs:
+            # We're stopping the environment
```

Comment:
It's not a bad state, we're getting here when we call `start` and then `stop` instead of just running the tests. I haven't found a great way to not do that.